lib/modules/platform/bitbucket/index.spec.ts

```diff
@@ -33,6 +33,7 @@ describe('modules/platform/bitbucket/index', () => {
     memCache = await import('../../../util/cache/memory');
     hostRules = jest.requireMock('../../../util/host-rules');
     bitbucket = await import('.');
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
```suggestion
```

Reply:
no, it's now required

Reply:
The PR had a warning about this line saying it's unused

Reply:
ok,will check again